command/please.go

```diff
@@ -118,9 +121,15 @@ func CmdPlease(c *cli.Context) (err error) {
 
 	// Prepare deploy script
 	var stdout strings.Builder
+	var stdoutBuf bytes.Buffer
 	cmd := exec.Command(deployScript, environment)
-	cmd.Stdout = &stdout
-	cmd.Stderr = os.Stderr
+	if logScript {
+		cmd.Stdout = io.MultiWriter(os.Stdout, &stdoutBuf)
```

Comment:
Let's just make this the default. It's probably surprising to not see the stdout output.